### DISCUSSION TITLE
Help understand the effect of convection length scale

### DISCUSSION THREAD

**Discussion by JulietteFranque** [firemodels/fds#13352] [Q&A] [answered]
Hi all,
I was seeing some unexpected behaviors in a more complicated simulation so I made a very simple case to reproduce. In this case, a steel object is placed inside the domain, everything is at ambient temperature, so "nothing happens". Given that, I believe that natural convection should apply. For a vertical plate, the user manual says Nu_free =  (0.825 + 0.324 Ra^1/6)^2. Since Ra = 0 (Tg - Twall = 0), Nu_free = 0.825^2 and h = k/L Nu_free. From my understanding, L corresponds to the parameter "CONVECTION_LENGTH_SCALE" which is by default 1 m. However, when I vary this parameter, I do not see any changes to the heat transfer coefficient (see DEVC 'H' in my file). When I vary the grid size, however, the heat transfer coefficient does change. Am I misunderstanding something about this "L" parameter? Is it dependent on the grid size?

I only uploaded my input file (I tried various  CONVECTION_LENGTH_SCALE and grid sizes). Let me know if you would like anything else. Thanks very much.
[steel_box.txt](https://github.com/user-attachments/files/16731426/steel_box.txt)

**Comment by mcgratta** [comment]
I'll take a look at it.

**Comment by mcgratta** [comment, marked as answer]
From The FDS Technical Reference Guide

![image](https://github.com/user-attachments/assets/ac12b370-f088-4a42-ac63-8f503191ab26)

You are hitting the 2*L/dn limit on the right of Eq. 7.14. We use this because these empirical correlations are not designed for the limit of zero velocity or zero temperature difference.

**Reply by JulietteFranque** [reply]
Thank you - I will have to see why the limit is hit in my "real" simulation... Is there an output that can tell me at which timesteps the limit is hit? Is the 'CONVECTIVE HEAT TRANSFER REGIME' designed for this? I'm not sure what the 'IMPACT' and 'RESOLVED' keywords refer to. Is there also a way to output Nu to a DEVC? I don't see that as an option

**Reply by mcgratta** [reply]
`IMPACT` is impinging jet and `RESOLVED` is DNS.

**Reply by mcgratta** [reply]
Nu is not an output
